nncf/quantization/advanced_parameters.py

```diff
@@ -276,13 +276,16 @@ class AdvancedAWQParameters:
     :type alpha_max: float
     :param steps: The number of the steps in grid search.
     :type steps: int
+    :param prefer_data_aware: Determines whether to use activations to calculate scales if activations are presented.
+    :type prefer_data_aware: bool
     """
 
     subset_size: int = 32
     percent_to_apply: float = 0.002
     alpha_min: float = 0.0
     alpha_max: float = 1.0
     steps: int = 100
+    prefer_data_aware: bool = True
```

Comment:
```suggestion
    :param use_data_aware_scaling: Whether to use activation data for scale calculation when available.
    :type use_data_aware_scaling: bool
    """

    subset_size: int = 32
    percent_to_apply: float = 0.002
    alpha_min: float = 0.0
    alpha_max: float = 1.0
    steps: int = 100
    use_data_aware_scaling: bool = True
```
What do you think about this?

Reply:
In my opinion, the word "prefer" conveys exactly what we want: when there's a dataset, preference will be given to the data-aware scenario.
@alexsu52  what do you think about `prefer_data_aware_scaling`?